lib/ansible/modules/system/iptables.py

```diff
@@ -460,6 +460,19 @@
     limit_burst: 20
     log_prefix: "IPTABLES:INFO: "
     log_level: info
+
+- name: Block container to visit 169.254.169.254
+  iptables:
+    action: insert
+    comment: prevent container to visit aws meta server
+    chain: FORWARD
+    protocol: tcp
+    destination_port: 80:443
```

Comment:
You first write that you block access to that IP, but then you only block access to a certain range of ports (80 to 443). Is there a reason you don't simply block all ports?

Reply:
From aws doc, it seems only 80/443 port is open on this meta data server, so I blocked 80/443 port, maybe block all ports is better